### PR TITLE
Rename Twiml API channel to Somleng

### PR DESCRIPTION
**Pull request by dwilkie**

### CODE REVIEW

File: temba/settings_common.py [hunk 1]
```diff
@@ -991,7 +991,7 @@
     "temba.channels.types.twilio_messaging_service.TwilioMessagingServiceType",
     "temba.channels.types.twilio_whatsapp.TwilioWhatsappType",
     "temba.channels.types.twilio.TwilioType",
-    "temba.channels.types.twiml_api.TwimlAPIType",
+    "temba.channels.types.somleng.SomlengType",
```
Comment (norkans7) [inline temba/settings_common.py:994]: we use alphabetical order here, so move that above

**Comment by dwilkie** [comment]
> poetry run ./code_check.py

I tried running this locally but got an error ```ModuleNotFoundError: No module named 'temba.settings'```

**Comment by dwilkie** [comment]
> @dwilkie you need to run `poetry run ./code_check.py` to run black, isort and ruff for code formatting checks

I got it working now and committed the changes

**Comment by norkans7** [comment]
@dwilkie tests need an update

**Comment by dwilkie** [comment]
> @dwilkie tests need an update

I'm not sure how to fix this test. I don't understand what it's doing.

**Comment by dwilkie** [comment]
> https://github.com/nyaruka/rapidpro/blob/4e12d0e2ab19515387d7f9d87e74e64e5fa20c3d/temba/channels/types/somleng/tests.py#L22
> 
> The test is checking the channel type is listed on the claim page, I think You renamed `TwiML` to `Somleng` then we can assert Somleng word is in the response so the type is listed on the channel claim page

https://github.com/nyaruka/rapidpro/pull/4546/commits/3bf53ed15d65738f0f0eeac1438e0693ac5b14b0

**Review by norkans7 — APPROVED** [review]
Looks good

File: templates/channels/types/twiml_api/claim.haml
Comment (rowanseymour) [inline templates/channels/types/twiml_api/claim.haml:1]: this path needs updated

**Review by rowanseymour — CHANGES_REQUESTED** [review]

**Comment by rowanseymour** [comment]
Gonna merge and fix...

**Pull request merged**